thoth/storages/data/alembic/versions/0c8eab5d4197_add_indexes_for_abi_compatability_.py

```diff
@@ -31,20 +31,22 @@ def upgrade():
     sa.ForeignKeyConstraint(['input_python_package_version_entity_id'], ['python_package_version_entity.id'], ondelete='CASCADE'),
     sa.PrimaryKeyConstraint('id')
     )
-    op.create_index('os_id_idx', 'package_extract_run', ['os_id'], unique=False)
-    op.create_index('os_version_id_idx', 'package_extract_run', ['os_id'], unique=False)
-    op.create_index('id_idx', 'python_package_version_entity', ['id'], unique=True)
-    op.create_index('python_artifact_id_idx', 'requires_symbol', ['python_artifact_id'], unique=False)
-    op.create_index('id_idx', 'versioned_symbol', ['id'], unique=True)
+    op.create_index('package_extract_run_os_id_idx', 'package_extract_run', ['os_id'], unique=False)
```

Comment:
We no longer require these indices based on #1569

Reply:
Sounds good. :+1:  Adjusted in 34ba4e2